pkg/satellite/service.go

```
@@ -235,18 +239,24 @@ func (s *Service) CreateProject(ctx context.Context, projectInfo ProjectInfo) (*
 		TermsAccepted: 1, //TODO: get lat version of Term of Use
 	}
 
-	// For now we make this operations sequentially.
-	// But soon we will add functionality to be able to
-	// do any operations in transaction scope
-	prj, err := s.store.Projects().Insert(ctx, project)
+	transaction, err := s.store.BeginTransaction(ctx)
 	if err != nil {
-		return nil, err
+		return nil, errs.New("Can not start transaction!")
```

Don't hide the error that happened actually happened.

```
return nil, Error.Wrap(err)
```